src/libraries/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -122,7 +122,7 @@ public enum ProtectionLevel
         public SslApplicationProtocol(byte[] protocol) { throw null; }
         public SslApplicationProtocol(string protocol) { throw null; }
         public System.ReadOnlyMemory<byte> Protocol { get { throw null; } }
-        public bool Equals(System.Net.Security.SslApplicationProtocol other) { throw null; }
+        public bool Equals([System.Diagnostics.CodeAnalysis.NotNullWhen(true)] System.Net.Security.SslApplicationProtocol other) { throw null; }
```

Comment:
I think you meant to put it on the line below

Reply:
(might be worth considering to add analyzer for cases like this)

Reply:
Oops, good catch, thanks.